core/driver/sources/src/main/scala/org/virtuslab/ideprobe/IntelliJFixture.scala

```diff
@@ -57,6 +57,11 @@ final case class IntelliJFixture(
     copy(plugins = plugin +: plugins)
   }
 
+  def withBundledCrossVersion(name: String, pluginVersion: String): IntelliJFixture = {
```

Comment:
This is most user facing api, perhaps we could move this setup somewhere else, to still be useful to us, but in more adequate place.
I thought maybe of something like putting it in InternalPlugins (the place I asked to cleanup from helper method with little use now).
```scala
object InternalPlugins {
  def installCrossVersionPlugin(name: String): IntelliJFixture => IntelliJFixture = {
    fixture => fixture.withPlugin(Plugin.BundledCrossVersion(name, fixture.version.compatibleScalaVersion, BuildInfo.version))
  }
}
```
It would be a useful utility function for us, as it would handle all versions at once. We don't need to care about being able to change version to something else, as all internal plugins will have the same version. 

In extension traits it can be used as:
```scala
  registerFixtureTransformer(InternalPlugins.installCrossVersionPlugin("ideprobe-pants"))
```